rustls/src/server/hs.rs

```diff
@@ -264,7 +264,7 @@ impl ExpectClientHello {
             #[cfg(feature = "tls12")]
             using_ems: false,
             done_retry: false,
-            send_ticket: false,
+            send_tickets: 0,
```

Comment:
There's a case where `ServerConfig::send_tls13_tickets` diverges from the various state `send_tickets`: where we're unwilling to resume with the client, because it doesn't support forward secrecy during resumption: https://github.com/rustls/rustls/blob/f7267730db165f867d3c3652e5fb0ee1d4d20bd7/rustls/src/server/tls13.rs#L308-L312